classification/assign-element-classification.js

```diff
@@ -0,0 +1,97 @@
+/*
+    This example demonstrates how assign classification to the element. The scenario is quite simple - there is hardcoded mapping between element name
+    and classification. I can be used as reference sample in case that more advanced logic is needed.
+    
+    It uses 2-legged authentication - this requires that application is added to facility as service.
+*/
+import { createToken } from '../common/auth.js';
+import { TandemClient } from '../common/tandemClient.js';
+import { ColumnFamilies, ColumnNames, MutateActions, QC } from '../common/utils.js';
+
+// update values below according to your environment
+const APS_CLIENT_ID = 'YOUR_CLIENT_ID';
+const APS_CLIENT_SECRET = 'YOUR_CLIENT_SECRET';
+const FACILITY_URN = 'YOUR_FACILITY_URN';
+
+async function main() {
+    // STEP 1 - obtain token to authenticate subsequent API calls
+    const token = await createToken(APS_CLIENT_ID,
+        APS_CLIENT_SECRET, 'data:read data:write');
+    const client = new TandemClient(() => {
+        return token;
+    });
+
+    // STEP 2 - get facility & facility template
+    const facilityId = FACILITY_URN;
+    const facility = await client.getFacility(facilityId);
+    const template = await client.getFacilityTemplate(facilityId);
+    // this defines mapping between element name and classification
+    const elementClassificationMap = {
+        'Door - Interior - Double' : 'Interior Doors',
+        'Door - Interior - Single' : 'Interior Doors',
+        'Door - Exterior - Double' : 'Exterior Doors',
+        'Door - Exterior - Single' : 'Exterior Doors'
+    };
+    // STEP 3 - build map between element name and classification id
+    const classificationMap = {};
```

Comment:
Maybe can use a `Map` here to be able to use cleaner existence checks with `.has`.

Reply:
Done